mesa_geo/raster_layers.py

```diff
@@ -415,6 +460,28 @@ def from_file(
             obj.apply_raster(values, attr_name=attr_name)
             return obj
 
+    def to_file(self, raster_file, attr_name: str = None, driver="GTiff"):
+        """Writes a raster layer to a file.
+        Args:
+            raster_file: Path to the raster file to write.
+            attr_name: Name of the attribute to write to the raster. If None, all attributes are written. Default is None.
+            driver: Driver to use for writing the raster. Default is "GTiff" (see GDAL docs at https://gdal.org/drivers/raster/index.html).
+        """
+        data = self.get_raster(attr_name)
+        with rio.open(
+            raster_file,
+            "w",
+            driver=driver,
+            width=self.width,
+            height=self.height,
+            count=data.shape[0],
+            dtype=data.dtype,
+            transform=self.transform,
+        ) as dataset:
+            dataset.write(data)
+            dataset.crs = self.crs
```

Comment:
What is the usage of the following 2 lines?

Reply:
Those 2 lines are a mistake... It is now fixed.